tests/data/state_estimation/leaf-without-power-sensor/params.json

```diff
@@ -0,0 +1,8 @@
+{
+  "calculation_method": ["newton_raphson", "iterative_linear"],
```

Comment:
since the timeline on solving this is not yet known because of the nature of the problem, i propose to mark these test cases as `fail` and merge, and instead open an issue on this to reflect the progress on solving the bug.

Reply:
done